### ISSUE TITLE
api extra information

### ISSUE THREAD

**Issue by bjeenie2** [danpaulson/swgoh.gg.public#215]
in the guild units api https://swgoh.gg/api/guilds/###/units/ it could be nice to have that the member who has the unit also specifies which zeta it has for that unit.

for example Zetas for CLS for a specific member: 

Zetas: [{name: "rebel manouvre", isLead: true},{name: "learn control", isLead: false},{name: "it binds all things", isLead: false}]

**Comment by danpaulson** [comment]
We have plans to expand the API but they are contingent on our refactor of the database.
